Essentials/src/com/earth2me/essentials/commands/Commandafk.java

```diff
@@ -59,25 +59,33 @@ private void toggleAfk(User sender, User user, String message) throws Exception
         }
         user.setDisplayNick();
         String msg = "";
+        String selfmsg = "";
         if (!user.toggleAfk()) {
-            //user.sendMessage(_("markedAsNotAway"));
             if (!user.isHidden()) {
                 msg = tl("userIsNotAway", user.getDisplayName());
+                selfmsg = tl("userIsNotAwaySelf");
             }
             user.updateActivity(false);
         } else {
-            //user.sendMessage(_("markedAsAway"));
             if (!user.isHidden()) {
                 if (message != null) {
                     msg = tl("userIsAwayWithMessage", user.getDisplayName(), message);
+                    selfmsg = tl("userIsAwaySelfWithMessage", message);
                 } else {
                     msg = tl("userIsAway", user.getDisplayName());
+                    selfmsg = tl("userIsAwaySelf");
                 }
             }
             user.setAfkMessage(message);
         }
         if (!msg.isEmpty()) {
-            ess.broadcastMessage(user, msg);
+            if (ess.getSettings().broadcastAfkMessage()) {
```

Comment:
Suggest changing to

```
if (ess.getSettings().broadcastAfkMessage())
    ess.broadcastMessage()...
user.sendMessage()
```

Reply:
This isn't consistent with EssentialsX's code style, which is generally to use blocks everywhere (though I admit I might have fallen foul of this in the past).

Reply:
Alright, add blocks to it - however the main point is that user.sendMessage is duplicated.

Reply:
I've updated both sections to remove the duplicated `sendMessage` usage.